source/nordic_sdk/components/libraries/bootloader_dfu/hci_transport/hci_mem_pool_internal.h

```diff
@@ -42,10 +42,10 @@
 #ifndef MEM_POOL_INTERNAL_H__
 #define MEM_POOL_INTERNAL_H__
 
-#define TX_BUF_SIZE       4u    /**< TX buffer size in bytes. */
-#define RX_BUF_SIZE       32u   /**< RX buffer size in bytes. */
+#define TX_BUF_SIZE       32u    /**< TX buffer size in bytes. */

```

Comment:
this should *not* be changed. I bet the bootloader won't work if you apply this change.